src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+$cell-size: 100px;
+$padding: 10px;
+$gap: 1px;
+$width: (($cell-size + $gap) * 7) - $gap;
+$days-of-week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+$background-color: #eee;
+$hover-color: #ffbfcb;
+
 body {
   margin: 0;
+  height: 100vh;
+  justify-content: center;
+  display: flex;
+  align-items: center;
+}
+
+.calendar {
+  width: $width;
+  padding: 10px;
+  gap: 1px;
+  display: flex;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid #000;
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: $hover-color;
+      transform: translateY(-20px);
+      transition-duration: 0.5s;
```

Comment:
Hover should be animated both on enter and on exit, can you think of a change to make here in order to achieve that?